src/main/java/org/jenkinsci/plugins/saml/SamlSecurityRealm.java

```diff
@@ -39,8 +39,8 @@
 import org.jenkinsci.plugins.saml.user.SamlCustomProperty;
 import org.kohsuke.stapler.*;
 import org.kohsuke.stapler.interceptor.RequirePOST;
-import org.pac4j.core.client.RedirectAction;
-import org.pac4j.core.client.RedirectAction.RedirectType;
+import org.pac4j.core.redirect.RedirectAction;
+import org.pac4j.core.redirect.RedirectAction.RedirectType;
 import org.springframework.dao.DataAccessException;
```

Comment:
Do you not plan to switch to Spring Security versions of Jenkins interfaces as well, now that you are on a baseline with JEP-227?

Reply:
sure, what is the replacement for DataAccessException?

Reply:
There is none.

Best to look at other PRs associated with JEP-227 for examples.